visualize_main.py

```diff
@@ -41,7 +41,7 @@ def visualize(options, data_file_name="out.json"):
 
 def make_temperature_plotter(data, dt):
   def plotter():
-    temperatures = data['temperature']
+    temperatures = data['Temperature']
     dt = 2
```

Comment:
shouldn't dt from the arguments be used instead of dt = 2?

Reply:
Didn't make this function. But now that you mention it, I don't see any reason why it should be as it is now ! Changing it! If @martinclason don't have anything to say about it?

Reply:
Fine by me! Might be I who added it. Probably just to test something.